2a_model/src/models/0_baseline_LSTM/config.yml

```diff
@@ -1,8 +1,8 @@
-exp_name: "0_baseline_LSTM"
+exp_name: 0_baseline_LSTM
```

Comment:
I specify that these should be quoted in the function that builds the config files so I was surprised to see that the quoted formatting gets lost when built on Tallgrass. I've played with this a bit before and I think if we updated the version of the `{yaml}` R package and added it to the container these config files would keep the quoted format as before. I've added a note in https://github.com/USGS-R/drb-do-ml/issues/158. 

It's good to know that the models still run even though these new config files don't have quotes around `exp_name` and `y_vars`.

Reply:
I think python will automatically read these as strings since they aren't numbers